src/plugins/charts/public/static/components/color_picker.scss

```diff
@@ -11,6 +11,11 @@ $visColorPickerWidth: $euiSizeL * 8; // 8 columns
     transform: scale(1.4);
   }
 
+  // removes the grey background that appears on the first dot when the popover is open
+  &:focus {
+    background-color: $euiColorEmptyShade;
```

Comment:
@myasonik does this raise any issues for you from an a11y perspective? I think, in this case, the focus is also on one of the color dots in the associated/open popover.

If this style overrides remains, we may want to just set it to a transparent background or none.

Reply:
This looks ok because we have that big circle around the color currently in use but, at the same time, this whole widget doesn't seem to be keyboard accessible (at least in Firefox) so I wasn't able to test how this looks like on buttons that aren't the currently selected color.